site/public/js/rainbow-customization.js

```diff
@@ -351,14 +351,105 @@ function addToTable() {
     cellDelete.appendChild(deleteLink);
 
     // Clear the form fields
-    document.getElementById('user_id').value = '';
+    document.getElementById('plagiarism_user_id').value = '';
     document.getElementById('g_id').value = '';
     document.getElementById('marks').value = '';
 }
 
+function getManualGrades() {
```

Comment:
This is almost duplication of plagiarism table code, instead of making a separate function, I think it is good practice to make it modular. Reduce redundancy, and  improve maintainability and integrity.

Reply:
[2790559](https://github.com/Submitty/Submitty/pull/10653/commits/2790559ef68d82acfd5af1d6dd32d62c9bdc2b48)